fixed_frame: world
camera_frame: ground_truth/husky/base
show_plots: true
lc_recent_pose_lockout_s: 20
lc_similarity_threshold: 0.55
lc_send_delay_s: 15
place_method: "salad"
keypoint_method: "ground_truth"
descriptor_method: "ground_truth"
pose_method: "ground_truth"
vlc_frame_period_s: 0.5
